Modules/Loadable/Markups/VTKWidgets/vtkSlicerPlaneRepresentation3D.cxx

```diff
@@ -184,6 +184,19 @@ void vtkSlicerPlaneRepresentation3D::UpdateFromMRML(vtkMRMLNode* caller, unsigne
   this->TextActor->SetTextProperty(this->GetControlPointsPipeline(controlPointType)->TextProperty);
 }
 
+//----------------------------------------------------------------------
+void vtkSlicerPlaneRepresentation3D::UpdateInteractionPipeline()
+{
+  Superclass::UpdateInteractionPipeline();
+
+  vtkMRMLMarkupsPlaneNode* planeNode = vtkMRMLMarkupsPlaneNode::SafeDownCast(this->GetMarkupsNode());
+  if (!planeNode || planeNode->GetNumberOfControlPoints() < 3)
+    {
+    this->InteractionPipeline->Actor->SetVisibility(false);
+    return;
+    }
```

Comment:
Should a call to `this->InteractionPipeline->Actor->SetVisibility(true);` be added below ?